src/TorchSharp/TorchVision/Image.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+using static TorchSharp.torch;
+
+namespace TorchSharp.torchvision
+{
+    public static partial class io
+    {
+
+        public static IImager DefaultImager { get; set; } = new ImagerSharp();
+
+        public enum ImageFormat
+        {
+            PNG,
+            JPEG,
+            UNKNOWN
```

Comment:
>UNKNOWN [](http://example.com/codeflow?start=12&length=7)

Maybe moving this as first entry will be better when adding more formats to this enum.